executor/oci/spec_unix.go

```diff
@@ -31,6 +35,15 @@ func GenerateSpec(ctx context.Context, meta executor.Meta, mounts []executor.Mou
 		ctx = namespaces.WithNamespace(ctx, "buildkit")
 	}
 
+	if system.SeccompSupported() {
+		if meta.SecMode == pb.SecMode_CONFINED {
+			opts = append(opts, seccomp.WithDefaultProfile())
+		}
+		if meta.SecMode == pb.SecMode_UNCONFINED {
+			opts = append(opts, entitlements.WithDefaultAdminProfile())
```

Comment:
This should add all caps, disable seccomp/apparmor completely (no seccompsupported check), remove maskpaths and readonly overrides and change sys mounts to rw.